### DISCUSSION TITLE
Auto-rerun Github Action workflow failure

### DISCUSSION THREAD

**Discussion by alexec** [community/community#33518] [Actions] [answered]
I have an action where one of the e2e test suits always fails. Re-running failed workflows usually fixes this. Rather than manually re-run, I would like them to automatically re-run.

**Comment by zarifpour** [comment, marked as answer]
Hey, maybe one of these could help:

- [Retry Step](https://github.com/marketplace/actions/retry-step)
- [Retry Action](https://github.com/marketplace/actions/retry-action)

**Reply by alexec** [reply]
This is what I used. Thank you!
